curvefs/src/space/space_main.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <brpc/server.h>
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include <memory>
+
+#include "curvefs/src/space/space_manager.h"
+#include "curvefs/src/space/space_service.h"
+
+namespace curvefs {
+namespace space {
+
+DEFINE_string(listenAddr, "0.0.0.0:19999", "space allocator service address");
+
+int Run(int argc, char* argv[]) {
+    google::ParseCommandLineFlags(&argc, &argv, true);
+
+    FLAGS_log_dir = "/home/wuhanqing/log";
+    google::InitGoogleLogging(argv[0]);
+
+    std::unique_ptr<SpaceManager> spaceManager(
+        new SpaceManagerImpl(SpaceManagerOption{}));
+    SpaceAllocServiceImpl spaceAllocService(spaceManager.get());
```

Comment:
Why use std::unique_ptr and get the raw pointer from smart pointer?    Use "SpaceManager spaceManager",   and passing "&spaceManager"?

Reply:
fix